### ISSUE TITLE
Merge bikeshed or die.

### ISSUE THREAD

**Issue by scheib** [WebBluetoothChrome/web-bluetooth#15]
https://github.com/WebBluetoothCG/web-bluetooth/pull/150 is a major spec refactor. Either a heavy merge of the implementation notes in https://github.com/WebBluetoothChrome/web-bluetooth needs to be done, or we should decide that it is not worth it.

What do you think @g-ortuno @jyasskin ?

**Comment by jyasskin** [comment]
My instinct is to take the implementation notes and re-apply them to the new spec, rather than trying to merge the new-spec over the implementation notes. This will also fix bad merges like #14.

**Comment by scheib** [comment]
Yes - re-apply is the only sane approach. This is sort of a call for "does the team want to continue with this implementation document approach?". So, what's the call on that? And if you're supportive, any chance you want to do the text shoveling, jyasskin@ ? :)

**Comment by jyasskin** [comment]
I can do the text shoveling. Whether it's useful is more a question for you and @g-ortuno. As the spec changes, I assume your merges into this document point out bits of the implementation that need to change. Has that been useful? If so, we should keep this.

**Comment by g-ortuno** [comment]
I think it is. Specially now that there have been so many small changes to the API since we implemented it e.g. throw error for no services.
